assets/constants.js

```diff
@@ -470,7 +470,9 @@ var constants = {
     "\\u001b", "\\u001c", "\\u001d", "\\u001e", "\\u001f"
   ],
 
-  "USE_NEW_SUGGESTION_FRAMEWORK": false,
+  "ENABLE_GCS_STORAGE_FOR_IMAGES": false,
```

Comment:
I don't understand, why is this here? It's been removed already in develop.

Reply:
Looks like I incorrectly merged. Removed!